src/ios/actions/rankScreenActions.js

```diff
@@ -36,3 +42,41 @@ export function fetchRivals() {
       });
   }
 }
+
+export function rivalVote(winner, loser) {
+  return function (dispatch) {
+    dispatch(rivalVoteInit())
+    //TODO: Calculate new winner and loser
+    
```

Comment:
yeah for now you can just use the formula on this https://en.wikipedia.org/wiki/Elo_rating_system#Mathematical_details

you calculate Ea and Eb for each dog and then you run the R'a = Ra + K(S - Ea) update respectively. S is 1 if win 0 if loss (Ea should be between 0 and 1).  We can just use K = 400 / (# of wins + # losses). lmk if this isn't clear and you need clarification